src/app/screens/LogIn/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { actionCreators } from '../../../redux/logIn/actions';
+
+import LogInForm from './layout.js';
+import './styles.css';
+
+class LogIn extends Component {
+  handleSubmit = values => {
+    this.props.dispatch(actionCreators.authUser(values.email, values.password));
+  };
+
+  render() {
+    return (
+      <div className="logInPage">
+        <h2 className="logInFormTitle">Tic-Tac-Toe Log In</h2>
```

Comment:
We should start to extract the strings into a `strings.js` file at the same level taht `index` `label` or `styles`

Reply:
this apply for all strings. This is because we have centralized strings into  each corresponding file and its easy to refactor. 
Also, in a future maybe we can apply translations

Reply:
Done